### PR TITLE
ZCS-1165 Universal UI - Fit and Finish of Appointment Scheduler

### PR DESCRIPTION
**Pull request by tarangkhandelwal**
Fixing:

1.  ZCS-1165: Universal UI - Fit and Finish of Appointment Scheduler
2.  Invite message view toolbar styling.

Changeset:
* Adding icons: ImgAttendeesOptional, ImgAttendeesRequired, ImgLocation, ImgPerson, ImgResource.
* ZmApptEditView.js: Adding function call to update scheduler date on date change.
* ZmFreeBusySchedulerView.js:
	* Added function updateSchedulerDate to update the start date of appointment inside the scheduler.
	* Added logic to show the scheduler view with appt start date on initialize.
	* Adding placeholder text for attendee input field.
* ZmMsg.properties: Adding string addAttendee for add attendee placeholder value inside scheduler.
* common.css,zm.css & skin.properties: Adding style rules to match the scheduler UX as per the new design.

### CODE REVIEW

File: WebRoot/templates/calendar/Appointment.template [hunk 1]
```diff
@@ -775,50 +775,54 @@
...
 		</table>
-		<table role="presentation" class='ZPropertySheet' cellspacing='6'>
+		<table role="presentation" class='ZPropertySheet ZmSchedulerLegendTable' cellspacing='6'>
 			<tr>
 				<td id='${id}_graphKeySpacer'>
 					<span id='${id}_showMoreLink' class="FakeAnchor" style='display:none;'><$= ZmMsg.showMore $></span>
 				</td>
 				<td><div class='ZmGraphKeyColorBox ZmScheduler-working'></div></td>
-				<td class='ZmGraphKeyColorText' style='padding-right:1em;'><$= ZmMsg.free $></td>
+				<td class='ZmGraphKeyColorText' style='padding:0 2em 0 1em;'><$= ZmMsg.free $></td>
```
Comment (kalpeshhpatel) [inline WebRoot/templates/calendar/Appointment.template]: Can we move these style rules to stylesheet?

File: WebRoot/css/zm.css [hunk 2]
```diff
@@ -3310,38 +3296,40 @@ HTML>BODY .appt-selected .appt_allday_body
...
 }
 
+.ZmSchedulerGridHeaderLeft {
+	@LeftSeparator@
+	@AlignLeft@
+	padding-left:@SkinViewGutterSpace@;
+}
+
+.ZmSchedulerGridHeaderLeft > div{
+	position:absolute;
```
Comment (kalpeshhpatel) [inline WebRoot/css/zm.css]: Don't we need other supporting property here? ie `top`, `left`.. etc?

Comment (tarangkhandelwal) [reply WebRoot/css/zm.css]: Not required, div is aligned based on its "relative" parent. Not specifically required a value here.

File: WebRoot/templates/calendar/Appointment.template (hunk 1 above)
Comment (tarangkhandelwal) [reply WebRoot/templates/calendar/Appointment.template]: There are style rules for: 
".ZPropertySheet.ZmSchedulerLegendTable>TBODY>TR>TD"
that I want to override in this case, so going for inline styling. 
Let me know if I should change the approach.

**Pull request merged**